### ISSUE TITLE
Chip input in form field doesn't show placeholder

### ISSUE THREAD

**Issue by jfcere** [angular/components#16380]
#### Feature Description

Add an optional _secondary placeholder_ for chips input component.

#### Use Case

In _Angular Material_ for _Angular.js_ chips input could have a _secondary placeholder_ for when one chip was already present in the input. IMHO this increases the user experience by letting him know that chips can be added.

![angular-material-chip-input-secondary-placeholder](https://user-images.githubusercontent.com/6987084/60116668-823c3700-9746-11e9-98d1-bcb1f089e344.gif)

Reference: https://material.angularjs.org/1.1.0/demo/chips

**Comment by Airblader** [comment]
Can't you just bind a placeholder depending on the number of chips that are already present?

**Comment by jfcere** [comment]
@Airblader yes of course but the placeholder would still appear on top of the input, not on the side where the user should click to type the new chip text and would not serve the same purpose.

Imagine having a list of email addresses, the placeholder on top of the list should not say "Enter new email" but preferably only "Emails". The purpose of the secondary tag is to invite the user to enter new values.

![image](https://user-images.githubusercontent.com/6987084/60121656-946fa280-9751-11e9-9cc7-90b8a3afb8b5.png)

**Comment by Airblader** [comment]
Yeah, sorry, I misunderstood.

**Comment by jfcere** [comment]
No worry, thanks for your suggestion... this only proves my request was not clear enough :)
